packages/flint/bin/index.js

```diff
@@ -4,6 +4,6 @@ import { enableCompileCache } from "node:module";
 
 enableCompileCache();
 
-await import("@flint.fyi/core/lib/ts-patch/install-patch.js");
+await import("@flint.fyi/core/ts-patch/install-patch");
```

Comment:
This touches a lot of the same files as #1102.
Would possibly be slightly simpler to do that first, but isn't blocking.